src/components/Templates/TemplatesList/List.tsx

```diff
@@ -0,0 +1,92 @@
+import type { Filters, Template } from '../types';
+import { useMemo, useState } from 'react';
+import { FaMagnifyingGlass } from 'react-icons/fa6';
+import { QuestionMark } from '@components/Icons';
+import { TemplateCard } from './TemplateCard';
+import { FiltersSection } from './FiltersSection';
+
+interface CardsProps {
+  templates: Template[];
+}
+
+export const List: React.FC<CardsProps> = ({ templates }) => {
+  const [filters, setFilters] = useState<Filters>({});
+
+  const filteredTemplates = useMemo(
+    () =>
+      templates.filter((template) => {
+        const matchesFramework = filters.framework
+          ? template.framework.name === filters.framework
```

Comment:
Is the name guaranteed?

```suggestion
          ? template.framework?.name === filters.framework
```

Reply:
it "should" because is enforced by the Template > Framework type. still you are right. a safeguard doesn't hurt